content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -38,7 +38,7 @@ different Kubernetes components.
 | `APIResponseCompression` | `false` | Alpha | 1.7 | |
 | `AppArmor` | `true` | Beta | 1.4 | |
 | `AttachVolumeLimit` | `false` | Alpha | 1.11 | |
-| `BlockVolume` | `false` | Alpha | 1.9 | |
```

Comment:
We should keep this line just like other features :)

Reply:
fixed !